docs/networks/arbitrum-sepolia.md

```diff
@@ -0,0 +1,111 @@
+# Testnet Configuration
+
+The Graph Network's testnet is on Arbitrum Sepolia (eip155:421614). Sepolia network information can be found at https://testnet.thegraph.com/explorer?chain=arbitrum-sepolia.
+
+## Latest Releases
+
+| Component       | Release                                                                         |
+| --------------- | ------------------------------------------------------------------------------- |
+| contracts       | [5.3.3](https://github.com/graphprotocol/contracts/releases/tag/v5.3.3)         |
+| indexer-agent   | [0.20.22](https://github.com/graphprotocol/indexer/releases/tag/v0.20.22)       |
+| indexer-cli     | [0.20.22](https://github.com/graphprotocol/indexer/releases/tag/v0.20.22)       |
+| indexer-service | [0.20.22](https://github.com/graphprotocol/indexer/releases/tag/v0.20.22)       |
+| graph-node      | [v0.33.0](https://github.com/graphprotocol/graph-node/releases/tag/v0.33.0)     |
+
+## Network Parameters
+
+| Parameter                   | Value             |
+| --------------------------- | ----------------- |
+| Epoch length                | ~ 2h (554 blocks) |
+| Maximum allocation lifetime | ~ 9h (4 epochs)   |
+
+## Contracts & accounts
+
+| Name               | Address                                                                                                                        |
+| ------------------ | ------------------------------------------------------------------------------------------------------------------------------ |
+| Graph Token (GRT ) | [`0xf8c05dCF59E8B28BFD5eed176C562bEbcfc7Ac04`](https://arbiscan.io/address/0xf8c05dCF59E8B28BFD5eed176C562bEbcfc7Ac04) |
```

Comment:
sorry if I wasn't clear, the link should be https://sepolia.arbiscan.io/address/0xf8c05dcf59e8b28bfd5eed176c562bebcfc7ac04

Same for other addresses on Arbitrum Sepolia.

Reply:
Ah yes, of course. 🤦